File: public/Lab-Files/Lesson08/Optional_Lab_8-1/hello.php
<?php 
$ua = $_SERVER['HTTP_USER_AGENT'];

{print"


<html>

<head>

<title>Browser detection with PHP</title>

</head>

<body>

<h1>
Hello, World!
</h1>

Your user agent is:
<strong>
{$ua}.
</strong>

</body>

</html>

";
}

?>

File: public/Lab-Files/Lesson08/Labs/Optional_Lab_8-1/hello.pl
#!/usr/bin/perl
use CGI qw/:all/;
$cgi_object = CGI::new();

print "Content-type: text/html\n\n";
print "<html>\n<head>\n<title>\nPerl CGI 
Example\n</title>\n<body>\n<h1>Hello, 
World!</h1>\nYour user agent is: <b>\n";
print $cgi_object->user_agent();
print "</b>.</html>\n";
